src/game/client/neo/c_neo_message.cpp

```diff
@@ -23,17 +27,42 @@ void C_NEO_Message::OnDataChanged(DataUpdateType_t updateType)
         return;
     }
 
-    if (Q_strlen(STRING(m_NetworkedMessageKey)) > 0)
+    if (V_strlen(m_NetworkedMessageKey) > 0)
```

Comment:
Instead of calculating the string length, you can check if the first character is non-zero (the null terminator), since this is faster: `if (*m_NetworkedMessageKey)`